src/main/java/com/segment/analytics/android/integrations/google/analytics/GoogleAnalyticsIntegration.java

```diff
@@ -333,46 +357,60 @@ private static int extractNumber(String text, int start) {
   }
 
   /** Send a product event. */
-  void sendProductEvent(String event, String category, Properties properties) {
+  void sendProductEvent(String event, String category, Properties eventProperties) {
```

Comment:
Is this just a semantic refactor here?

Reply:
yeah renamed since there are now three properties we deal with (event properties, product properties and the combined properties)